client/components/payments-activity/payments-activity-data.tsx

```diff
@@ -10,17 +11,43 @@ import HelpOutlineIcon from 'gridicons/dist/help-outline';
  */
 import PaymentsDataTile from './payments-data-tile';
 import { ClickTooltip } from '../tooltip';
+import { usePaymentActivityData } from 'wcpay/data';
 import { getAdminUrl } from 'wcpay/utils';
+import type { DateRange } from './types';
+
 import './style.scss';
 
+/**
+ * This will be replaces in the future with a dynamic date range picker.
+ */
+const getDateRange = (): DateRange => {
+	return {
+		// Subtract 7 days from the current date.
+		date_start: moment()
+			.subtract( 7, 'd' )
+			.format( 'YYYY-MM-DD\\THH:mm:ss' ),
+		date_end: moment().format( 'YYYY-MM-DD\\THH:mm:ss' ),
+	};
+};
+
 const PaymentsActivityData: React.FC = () => {
+	const { paymentActivityData, isLoading } = usePaymentActivityData(
+		getDateRange()
+	);
+
+	const totalPaymentsVolume = paymentActivityData?.total_payment_volume ?? 0;
+	const charges = paymentActivityData?.charges ?? 0;
+	const fees = paymentActivityData?.fees ?? 0;
+	const disputes = paymentActivityData?.disputes ?? 0;
+	const refunds = paymentActivityData?.refunds ?? 0;
```

Comment:
💭 This is a minor comment and we may not agree, but perhaps we should be more explicit with the naming of these variables.

For example: `disputes` could mean number of disputes. `disputesAmount` or `disputesTotal` would add clarity.

Happy to leave as is also.